sortinghat/server/sortinghat_admin.py

```diff
@@ -140,14 +144,86 @@ def upgrade(no_database):
     click.secho("SortingHat upgrade completed", fg='bright_cyan')
 
 
-def _create_database():
+@click.command()
+@click.option('--no-interactive', is_flag=True, default=False,
+              help="Run the command in no interactive mode.")
+def migrate_old_database(no_interactive):
+    """Migrate SortingHat 0.7 database schema to 0.8 and all the data"""
+
+    import MySQLdb
+    from django.conf import settings
+    from .utils.create_sh_0_7_fixture import create_sh_fixture
+
+    def _database_table_exists(db_params, table):
+        try:
+            MySQLdb.connect(
+                user=db_params['USER'],
+                password=db_params['PASSWORD'],
+                host=db_params['HOST'],
+                port=int(db_params['PORT']),
+                database=db_params['NAME']
+            ).cursor().execute(
+                f"SELECT * FROM {table} LIMIT 0;"
+            )
+            return True
+        except MySQLdb.DatabaseError as exc:
+            click.echo(exc)
+            return False
+
+    def _backup_tables(db_params, from_db, to_db):
+        try:
+            cursor = MySQLdb.connect(
+                user=db_params['USER'],
+                password=db_params['PASSWORD'],
+                host=db_params['HOST'],
+                port=int(db_params['PORT'])
+            ).cursor()
+
+            cursor.execute(f"SHOW TABLES FROM {from_db};")
+            for (table,) in cursor:
+                click.echo(f'Backup table {table}')
+                cursor.execute(
+                    f"RENAME TABLE {from_db}.{table} TO {to_db}.{table};"
+                )
+        except MySQLdb.DatabaseError as exc:
+            msg = f"Error in backup database '{from_db}': {exc}."
+            raise click.ClickException(msg)
+
+    db_params = settings.DATABASES['default']
+
+    click.secho("Migrate SortingHat database if schema version < 0.8 ...",
```

Comment:
I think you should move this under the `if` statement, so it's only shown when the migration is taking place.